.github/workflows/release.yml

```diff
@@ -27,6 +27,6 @@ jobs:
       - name: Run cockpituous
         run: |
           # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
-          export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+          export HOME=$(getent passwd $(id -u) | cut -f6 -d:) DOWNLOAD_DIST_OPTIONS=--wait
```

Comment:
I'd prefer this on a separate line to make it easier to read, but not a blocker.